tools/qemu_dev_start

```diff
@@ -216,7 +216,7 @@ if [ -n "$_arg_enable_wan" ]; then
     fi
 
     # DHCP server for WAN ifc
-    dnsmasq -F 172.99.0.100,172.99.0.100 --dhcp-option=3,172.99.0.1 -i "$WAN_IFC" --dhcp-authoritative --log-dhcp
+    dnsmasq -F 172.99.0.100,172.99.0.100 --dhcp-option=3,172.99.0.1 -i "$WAN_IFC" --dhcp-authoritative --log-dhcp --port=5353 --bind-dynamic
```

Comment:
Are you sure DNS server for virtual machines will work as is without any change if the dnsmasq is listening on port 5353 ?